### PR TITLE
Faq fix

### PR DESCRIPTION
**Pull request by cloveras**

### CODE REVIEW

File: report-api-faq.md [hunk 1]
```diff
@@ -125,73 +99,36 @@ as there are no settlements being made in that environment. The data that
...
-   Periodic checks of settlements would thus not make sense until the merchant has a positive again.
-1. Merchants have different risk profiles (or could have at least). If we deemed that we would hold
-   back some of the balance from payouts to an airline company.
-   Let's say they sell three tickets at 5000 NOK each. We then change their risk profile saying we
-   need to hold back 7500 NOK and then the settlement run. We will then pay out 7500 NOK
-   (if there are no fees). Now: What captures will then be part of that settlement?
-1. We may, in rare cases, manually correct payouts.
+**Important:** It is not yet possible to use the Report API to retrieve data
+for *Vippsnummer* sale units. The Report API can only be used to sale units
+that have API access (which *Vippsnummer* sale units do not have.)
```
Comment (Haldormyre) [inline report-api-faq.md:115]:
```suggestion
**Important:** It is not yet possible to use the Report API to retrieve data
for *Vippsnummer* sales units. The Report API can only be used to fetch data for sales units
that have API access (which *Vippsnummer* sales units do not have.)
```

File: report-api-faq.md [hunk 2]
```diff
@@ -244,48 +204,96 @@ and configuration is not yet generally available.
...
+There are several reasons for this, including:
+
+1. A merchant can have a negative balance and not get settlement payouts for a long time.
+   Periodic checks of settlements would thus not make sense until the merchant has a positive again.
+1. Merchants have different risk profiles (or could have at least). If we deemed that we would hold
+   back some of the balance from payouts to an airline company.
+   Let's say they sell three tickets at 5000 NOK each. We then change their risk profile saying we
+   need to hold back 7500 NOK and then the settlement run. We will then pay out 7500 NOK
+   (if there are no fees). Now: What captures will then be part of that settlement?
+1. We may, in rare cases, manually correct payouts.
```
Comment (Haldormyre) [inline report-api-faq.md:275]:
```suggestion
1. A merchant can have a negative balance and not get settlement payouts for a long time.
   Periodic checks of settlements would thus not make sense until the merchant has a positive again.
2. Merchants have different risk profiles (or could have at least). If we deemed that we would hold
   back some of the balance from payouts to an airline company.
   Let's say they sell three tickets at 5000 NOK each. We then change their risk profile saying we
   need to hold back 7500 NOK and then the settlement run. We will then pay out 7500 NOK
   (if there are no fees). Now: What captures will then be part of that settlement?
3. We may, in rare cases, manually correct payouts.
```

**Review by matsjohans — APPROVED** [review]
lgtm

File: report-api-faq.md [hunk 3]
```diff
@@ -125,73 +99,36 @@ as there are no settlements being made in that environment. The data that
...
-   Periodic checks of settlements would thus not make sense until the merchant has a positive again.
-1. Merchants have different risk profiles (or could have at least). If we deemed that we would hold
-   back some of the balance from payouts to an airline company.
-   Let's say they sell three tickets at 5000 NOK each. We then change their risk profile saying we
-   need to hold back 7500 NOK and then the settlement run. We will then pay out 7500 NOK
-   (if there are no fees). Now: What captures will then be part of that settlement?
-1. We may, in rare cases, manually correct payouts.
+**Important:** It is not yet possible to use the Report API to retrieve data
+for *Vippsnummer* sales units. The Report API can only be used to fetch data for sales units
+that have API access (which *Vippsnummer* sales units do not have.)
```
Comment (Haldormyre) [inline report-api-faq.md:115]: Should we note something here about it not being the case when using accounting partners?

Comment (cloveras) [reply report-api-faq.md:115]: Hmm, maybe, but it's mentioned several other places.

**Pull request merged**